### PR TITLE
Fixed bug #39 where text and margins where incorrect on mobile devices.

### PR DESCRIPTION
**Pull request by WEREMSOFT**
Also fixed indentation, structural problems and missed template changes.

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -15,3 +15,9 @@ npm run serve
 npm run build
 ```
 
+# Important
+
+If you are using parcel to work with the site("npm run serve"), the site wil NOT auto update when you change a file. This is because a bug in parcel that prevents it to dettect changes when the containing folder have dots (like in inaka.github.io). To fix it, rename the folder to inaka-github-io.
+
+The changes will not be shown on the site until you *merge to master*. The default branch is dev. Keep that in mind if you are maintaining the site.
```
Comment (elbrujohalcon) [inline README.MD]: This is not true anymore. Now the default branch is `master`.

File: templates/index.template.html [hunk 2]
```diff
@@ -66,12 +66,11 @@ <h2 class="display-3 text-white">Open-source enthusiasts</h2>
         </div>
         <div class="col-md-8 order-1">
           <div class="p-5">
-            <h2 class="display-4">Who we are</h2>
-            <p>Inaka is a community of open-source enthusiasts (developers, PMs, devops, QA experts, etc.).</p>
-            <p>We maintain all the repos listed below (and some others, too).</p>
-          </div>
+              <h2 class="display-4">Who we are?</h2>
```
Comment (elbrujohalcon) [inline templates/index.template.html]: The `?` was intentionally removed before.

File: templates/index.template.html [hunk 3]
```diff
@@ -106,8 +109,8 @@ <h2 class="display-4">Proud of our history</h2>
         <div class="col-md-8 order-1">
           <div class="p-5">
             <h2 class="display-4">Contact Us</h2>
-            <p>If you have questions, suggestions or comments about any of these projects (or if you want to join us) please contact the dev team
-              at <a href="mailto:inakos@googlegroups.com">inakos@googlegroups.com</a></p>
+            <p>If you have questions, suggestions or comments about any of these projects please contact the dev team
```
Comment (elbrujohalcon) [inline templates/index.template.html]: I think the text and anchor were worth having. Why removing them?

File: templates/index.template.html (hunk 2 above)
Comment (WEREMSOFT) [reply templates/index.template.html]: There was some mistake regarding some changes on the template and in the index.html. It should be ok now.

File: templates/index.template.html (hunk 3 above)
Comment (WEREMSOFT) [reply templates/index.template.html]: There was a wrong correction on the template. It should be ok now.

**Pull request merged**